index.html

```diff
@@ -1,192 +1,192 @@
-            If you like what you see and have a project you need coded,
-            don’t hestiate to contact me.</p>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Dany's Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <!-- <link rel="stylesheet" type="text/css" media="screen and (max-device-width: 480px) and (resolution:163dpi)" href="shetland.css"/> -->
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <header>
+            <a href="#"class="Logo">My Logo</a>
+            <nav class="Navbar">
+                <div class="menu"><a href="#"><img src="img/humburger-btn.svg" alt="menu"></a></div>
+            </nav>   
+        </header>
+        <section>
+                <h1 class=Perso">My name is Amiral<br> I am glad to meet you!</h1>
+                <p class="bio"> I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
 
-            <h2 class="connect">LET'S CONNECT</h2>
-            <ul class="icon-bio">
-                <li><a href="www.twitter.com/amiralda"><img src="img/twitter.svg" alt="twitter-icon"></a></li>
-                <li><a href="https://www.linkedin.com/danyaugustin/"><img src="img/linkedin.svg" alt="linkedin-icon"></a></li>
-                <li><a href="https://github.com/amiralda"><img src="img/github.svg" alt="github-icon"></a></li>
-                <li><a href="https://www.medium.com/amiralda/"><img src="img/medium.svg" alt="medium-icon"></a></li>
-                <li><a href="https://angel.co/u/dany-augustin"><img src="img/angellist.svg" alt="angellist-icon"></a></li>
-            </ul>
-    </section>
+                <h2 class="connect">LET'S CONNECT</h2>
+                <ul class="icon-bio">
+                    <li><a href="https://twitter.com/amiraldany"><img src="img/twitter.svg" alt="twitter-icon"></a></li>
+                    <li><a href="https://www.linkedin.com/in/dany-augustin/"><img src="img/linkedin.svg" alt="linkedin-icon"></a></li>
+                    <li><a href="https://github.com/amiralda"><img src="img/github.svg" alt="github-icon"></a></li>
+                    <li><a href="https://medium.com/@danyaugustin1982"><img src="img/medium.svg" alt="medium-icon"></a></li>
+                    <li><a href="https://angel.co/u/dany-augustin"><img src="img/angellist.svg" alt="angellist-icon"></a></li>
+                </ul>
+        </section>
```

Comment:
- [ ] No doubt, you have made a brilliant effort to drive this project to this point, and now, there's only a few changes that I think needs to be done to match requirement and make your portfolio more appealing by checking the following tips:
  - [ ] Kindly ensure that you use the spacing elements and units as recommended by the Figma design. Elements like `padding`, `margin`, `gap`, `height`, `width`, etc.
  - [ ] Also, your usage of fonts, kindly stick to the figma's recommendation. You might have to download some of those webfonts from [Google Fonts](fonts.google.com) and import it into your file :thinking: 
  - [ ] The menu button for the mobile screen is still visible at a min-width 767px, and this is good for user experience. You have enough space to display your navigation bar now, unlike small screens.

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/222829712-019f78fd-2d82-427c-9c60-2a425f76b735.png)|![image](https://user-images.githubusercontent.com/99715640/222829856-c898ef7a-a7ea-42f3-bfd6-4c5efccea009.png)|

**_Overall, kindly make your design look very identical to the figma design_**

Reply:
here you can use something like:
  max-width: clamp(30ch, 40ch, 50ch);
  
  Or a simpler version:
  max-width: 45ch;

Reply:
But the BG image need ajustment:
Your can try:
Overflow hidden property.